etc/scripts/redfish-update.in

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
maybe?!
```suggestion
#!/bin/sh
```
If so, also in [extensions/redfish-config](https://github.com/openSUSE/wicked/pull/909/files#diff-60494f4204af41e31c66ff5fadda031cc7893cc49a37d01c0927ec06dba4b048)

Reply:
All our scripts are using bash, no need to trigger to require a further sh implementation. It can be changed when we port all extensions to not use bash features. Out of scope for now and not part of the redfish support feature request.